erpnext_com/www/docs/user/manual/en/setting-up/email/email-dropbox.md

```diff
@@ -0,0 +1,27 @@
+<!-- add-breadcrumbs -->
+
+# Email Dropbox
+
+When you receive emails from leads, customers, suppliers and other business partners, you may like to attach that to relevant records in ERPNext. You can save the email and upload it as an attachment. However, this is a cumbersome process. ERPNext offers a better way to handle this.
```

Comment:
I think "want to" would sound better than "like to" in the first line.

Reply:
something along the lines of "you might want to" would be better, no?

Reply:
How does this sound:

When you receive emails from leads, customers, suppliers, or other business partners, you might wish to attach those to their relevant records in ERPNext. One way to do so is to save the email and upload it as an attachment. However, this could turn into a cumbersome process. ERPNext offers a better way to handle this through its "Email Dropbox" feature.

When enabled, each record in ERPNext will have it's own unique email address to which you can forward emails. ERPNext then automagically attaches the email to that specific record.

Reply:
I have made changes as per your suggestion. Replaced automagically with automatically.